### DISCUSSION TITLE
[@types/react-dom] version 18.3.3 doesn't include canary or experimental types

### DISCUSSION THREAD

**Discussion by bd452** [DefinitelyTyped/DefinitelyTyped#71398] [Issues with a @types package]
Version 18.3.3 of `@types/react-dom` doesn't include the canary or experimental type definitions, though 18.3.1 does. 

The default behavior of npm installations allowing upgrades of minor versions means that anyone requiring `^18.3.1` will get types that aren't backwards-compatible with the previous version. 

This should be moved either to a new version (since it's a breaking change) or fixed to include the missing types.

There is also no npm package for `react-dom@18.3.3`

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment]
Should be fixed with latest `@types/*` i.e. `@types/react@18.3.15` and `@types/react-dom@18.3.5`
